gradescope_utils/autograder_utils/json_test_runner.py

```diff
@@ -68,13 +71,17 @@ def buildResult(self, test, err=None):
         tags = self.getTags(test)
         number = self.getNumber(test)
         visibility = self.getVisibility(test)
+        hide_errors = self.getHideErrors(test)
         score = self.getScore(test)
         if score is None:
             score = weight if passed else 0.0
 
         output = self.getOutput()
         if err:
-            output += "Test Failed: {0}\n".format(err[1])
+            if hide_errors:
+                output += "Test Failed: Error Message Hidden\n"
```

Comment:
I think that if we're not showing the assertion failure message, we don't really need to say that anything was hidden because it's not really relevant information for the students.


```suggestion
                output += "Test Failed.\n"
```

Reply:
To be honest, it might even be okay to not add anything to the output in this case, and rely on any other output to convey the test failure more explicitly, but I think this is fine for now.

Another idea is that rather than taking in a boolean flag, the decorator could take in a string that would be used in the scenario where tests are hidden. It could have a default of "Test Failed" but could be customized by the developer.